exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/data/data_customer_support.ipynb

```diff
@@ -0,0 +1,204 @@
+{
```

Comment:
<p>I think you can load the dataset straight into pandas</p><p><code>df = kagglehub.dataset_load(KaggleDatasetAdapter.PANDAS,&nbsp;"uraj520/customer-support-ticket-dataset")</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/364/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>